pom.xml

```diff
@@ -162,6 +190,24 @@
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>rpm-maven-plugin</artifactId>
                     </plugin>
+                    <plugin>
```

Comment:
You are including the `rpm-maven-plugin` twice now. If you build the project from the command line you will see that this leads to a warning:
```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for nl.knaw.dans.easy:easy-download:jar:1.1.3-SNAPSHOT
[WARNING] 'profiles.profile[rpm].plugins.plugin.(groupId:artifactId)' must be unique but found duplicate declaration of plugin org.codehaus.mojo:rpm-maven-plugin @ line 193, column 29
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```
Remove the first declaration of `rpm-maven-plugin`.

Reply:
done